tasks/exec.cr

```diff
@@ -12,21 +12,25 @@ class Lucky::Exec < LuckyCli::Task
     setting template_path : String = "#{__DIR__}/exec_template.cr.template"
   end
 
+  def print_help_or_call(args : Array(String), io : IO = STDERR)
```

Comment:
Nice catch. Could you add  a comment about why this is here so people in the future know what's up?

Reply:
Good idea. Done.